components/raftstore/src/store/snap.rs

```diff
@@ -66,8 +66,8 @@ const DEL_RANGE_PREFIX: &str = "del_range";
 
 const TMP_FILE_SUFFIX: &str = ".tmp";
 const SST_FILE_SUFFIX: &str = ".sst";
-const CLONE_FILE_SUFFIX: &str = ".clone";
-const META_FILE_SUFFIX: &str = ".meta";
+pub const CLONE_FILE_SUFFIX: &str = ".clone";
```

Comment:
You can call `get_total_snap_size` directly.

Reply:
changed to reuse the pd.rs store heartbeat logic.

Reply:
Why not remove `pub`?

Reply:
The current implementation version no longer needs to use the global variable, just remove pub and keep the original non-pub mode

Reply:
removed